tests/integration/test_jobs.py

```diff
@@ -71,3 +71,24 @@ def test_last_job_runs(w):
 
     for line in sorted(summary, key=lambda s: s['last_finished'], reverse=True):
         logging.info(f'Latest: {line}')
+
+
+def test_create_job():
+
+    from databricks.sdk import JobsAPI
+    from databricks.sdk.core import ApiClient
+    from databricks.sdk.service.jobs import (BaseClusterInfo, JobCluster,
+                                             JobTaskSettings, PythonWheelTask)
+
+    client = ApiClient()
+    api = JobsAPI(client)
```

Comment:
why do you initialize clients directly and not using the provided `w` fixture, that takes care of local debugging when necessary?

Reply:
didn't know of this fixture - will adapt 👍🏻